# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/FetchEventOpProxyParent.cpp
# Commit: 6cbb442f4772
# Full Hash: 6cbb442f477260469cab9f1d81e33fcaede82029
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-18 09:51:31
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r dbdca4661a35 -r 6cbb442f4772 dom/serviceworkers/FetchEventOpProxyParent.cpp
--- a/dom/serviceworkers/FetchEventOpProxyParent.cpp	Tue Jan 17 15:13:45 2023 +0000
+++ b/dom/serviceworkers/FetchEventOpProxyParent.cpp	Tue Jan 17 15:13:46 2023 +0000
@@ -160,7 +160,7 @@
         RemoteLazyInputStream::WrapStream(aBodyStream);
     MOZ_DIAGNOSTIC_ASSERT(stream);
 
-    copyRequest.body().ref().get_ParentToChildStream().stream() = stream;
+    copyRequest.body().ref().get_ParentToChildStream() = stream;
   }
 
   Unused << aManager->SendPFetchEventOpProxyConstructor(actor, copyArgs);